udm/beginner/party.md

```diff
@@ -59,6 +59,11 @@ Sample JSON Data
       "infoString": "123 Elm Street, Springfield, 12345, USA"
     }
   ],
+  "TelecomNumber" : {
```

Comment:
I guess there is no need to store the telecom number and postal address in the 'infostring' field in the ContactMech entity, as there is a separate entity for storing them. The 'infostring' should be used only for storing email or similar information.